release/release.md

```diff
@@ -0,0 +1,112 @@
+# How to do a Kata Containers Release
+
+Hi if you are reading this document you may also want to create a kata Containers Release.
+
+The Kata Containers Release Process is defined in the follwoing documents.
+
+https://github.com/kata-containers/documentation/blob/master/Releases.md
+
+To simply the process of read each release we have created a checklist for it.
+
+https://github.com/kata-containers/documentation/blob/master/Release-Checklist.md
+
+In order to simplify the process of do the Release Checklist we have semi-automated most of the process.
+
+So lets get started.
+
+## Requeriments 
+- Go (https://golang.org/doc/install#)
+
+- hub (https://github.com/github/hub)
+  If you are using github 2FA, configure hub manually creating a file `~/.config/hub`
+  ```yaml
+  github.com:
+  - user: YOUR_USER
+  oauth_token: TOKEN
+  protocol: https
+  ```
+  - OBS account with permissions on /home:katacontainers (https://build.opensuse.org/project/subprojects/home:katacontainers)
```

Comment:
I think you may have indented this and subsequent bullets, when they are intended to be at the same level as the previous one - it looks odd when rendered - please check.